Cargo.toml

```diff
@@ -19,12 +19,12 @@ bitcoin_support=["bitcoin","serde","serde_cbor"]
 
 [dependencies]
 rand="0.7"
-byteorder="1.2"
-lru-cache = "0.1.1"
-bitcoin_hashes = "0.7"
-bitcoin = { version= "0.21", features=["serde"], optional = true }
+byteorder="1.3"
+lru-cache = "0.1.2"
+bitcoin_hashes = "0.9"
+bitcoin = { version = "0.25", features=["use-serde"], optional = true }
 serde = { version ="1", optional = true }
-serde_cbor = { version="0.10", optional = true }
+serde_cbor = { version="0.11", optional = true }
 
 [dev-dependencies]
-hex = "0.3"
+hex = "0.4"
```

Comment:
You can get rid of this dependency like it was done for rust-bitcoin (https://github.com/rust-bitcoin/rust-bitcoin/pull/482/commits/07fb14b8872507c40e483c5524e72067c372f2db) and use `Vec::from_hex` instead